draft-ietf-curdle-gss-keyex-sha2.xml

```diff
@@ -364,7 +363,7 @@
         </figure>
 
         <t>This value is called the exchange hash, and it is used to
-        authenticate the key exchange. The exchange hash SHOULD be kept
+        authenticate the key exchange. The exchange hash MUST be kept
```

Comment:
I'm not so sure about this change...
RSA signature of this hash would expose it to an attacker
and it is defined as such (SHOULD, not MUST) in the original SSH RFC...

Reply:
Fixed